post/release-0-2.qmd

```diff
@@ -56,19 +147,8 @@ four other options (`"vincenty"`, `"thomas"`, `"andoyer"`, and `"karney"`) were
 added to ensure GeoArrow can faithfully represent the intent of an arbitrary Parquet
 and/or Iceberg source.
 
-Third, the GeoArrow 0.2 specification formalizes two types that had been used
-internally in the [GeoArrow Rust implementation](https://github.com/geoarrow/geoarrow-rs):
-
-- The "box" memory layout and corresponding `geoarrow.box` extension name represents
-  min/max values of X, Y, [Z, [M]], represented as a struct similar to a point with
-  separated coordinates.
-- The "geometry" and "geometrycollection" memory layouts and corresponding `geoarrow.geometry`
-  and `geoarrow.geometrycollection` extension names allow arbitrary geometries to be
-  represented within a single feature and within a single array using a union of the other
-  native encodings.
+## Implementations
```

Comment:
I'm kinda meh about having the implementation section here. Or at least I want to have it exist in a way that doesn't get quickly out of date.

I.e. hopefully we won't have another spec release for a while, but I don't want people a year or two from now to refer to this implementations section frozen in time.

Perhaps even we should have two separate blog posts? One to announce the new 0.2 spec release and another to promote implementations?

Reply:
> I'd rather only share + promote one link and the spec additions, while cool for low-level developers, are also very dry and have a very small audience

That's fair. Maybe we should just have a sentence when we get to the implementations section "this content is valid as of May 2025" and implementations may have progressed since then.